packages/anthropic/src/anthropic-messages-language-model.ts

```diff
@@ -179,21 +183,32 @@ export class AnthropicMessagesLanguageModel implements LanguageModelV1 {
     }
 
     return combineHeaders(
-      this.config.headers(),
+      await resolve(this.config.headers),
       betas.size > 0 ? { 'anthropic-beta': Array.from(betas).join(',') } : {},
       headers,
     );
   }
 
+  private buildRequestUrl(isStreaming: boolean): string {
+    return (
+      this.config.buildRequestUrl?.(this.config.baseURL, isStreaming) ??
+      `${this.config.baseURL}/messages`
+    );
+  }
+
+  private transformArgs(args: Record<string, any>): Record<string, any> {
```

Comment:
`transformArgs` had me wondering at first what it does. Based on the usage below, I'm wondering if `transformBody` or `transformRequestBody` would be a more intuitive name for both this method and the config property?

Reply:
Agreed, renamed to `transformRequestBody`.